analytics/src/test/java/brocoeur/example/analytics/service/UserWinLossByGameServiceTest.java

```diff
@@ -33,16 +31,15 @@ void shouldUpdateAnalyticAccordingToWinOrLoss() {
         // Given
         var gameId = 123;
         var userId = 8;
-        var listOfIsWinner = List.of(true, false, false);
         var amount = 100;
         var linkedJobId = 789456;
-        var playerResponse = new PlayerResponse(gameId, userId, listOfIsWinner, amount, linkedJobId);
+        var playerResponse = new PlayerResponse(gameId, userId, 50, amount, linkedJobId);
```

Comment:
What does 50 represents ? Wouldn't it be better to have in a var with a meaningful name ?

Reply:
this is a test, you can find what it represents in the 'PlayerResponse' constructor.